docs/de-DE/guide/README.md

```diff
@@ -134,7 +134,7 @@
    ```
 
 
-   ##### Von Quellcode auf [crates.io](https://crates.io/):
```

Comment:
This link is intentionally meant to link to the home page of Crates.io, so users can learn about the platform if they aren't familiar with it.